samples/subapp2-basic/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "subapp2-basic",
+  "version": "2.0.2",
+  "description": "Web application using Electrode X with React v18",
+  "homepage": "",
+  "scripts": {
+    "dev": "xrun -q electrode/dev",
+    "test": "xrun electrode/check",
+    "build": "xrun electrode/build",
+    "start": "node lib/server"
+  },
+  "author": {
+    "name": "Mateja Ristic",
+    "email": "mateja.ristic@gmail.com"
+  },
+  "contributors": [],
+  "main": "lib/server/index.js",
+  "keywords": [
+    "electrode",
+    "web"
+  ],
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "license": "UNLICENSED",
+  "engines": {
+    "node": ">= 12",
+    "npm": ">= 6"
+  },
+  "prettier": {
+    "printWidth": 100
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.12.5",
+    "@xarc/app": "../../packages/xarc-app",
+    "@xarc/fastify-server": "^3.3.0",
+    "@xarc/react": "../../packages/xarc-react",
+    "@xarc/react-query": "../../packages/xarc-react-query",
+    "@xarc/react-redux": "../../packages/xarc-react-redux",
+    "@xarc/react-router": "../../packages/xarc-react-router"
+  },
+  "devDependencies": {
+    "@types/node": "^14.14.6",
+    "@xarc/app-dev": "../../packages/xarc-app-dev",
+    "@xarc/run": "1.1.1",
+    "@xarc/opt-postcss": "../../packages/xarc-opt-postcss",
```

Comment:
can we follow same pattern as in other samples and add fyn dependencies with relative url